labshare-client/src/localization.js

```diff
@@ -9,10 +9,50 @@ export let localization = {
   silentFallbackWarn: true,
   messages: {
     en: {
+      backend: {
+        formValidation: {
+          // english should go below here
+          invalid_email: "Invalid e-mail adress",
+          invalid_firstname: "Invalid First Name (max. 200 characters)",
+          invalid_lastname: "Invalid Last Name (max. 200 characters)",
+          invalid_zipcode: "Invalid zipcode (should be 5 digits)",
+          invalid_city: "Invalid city (max. 200 characters)",
+          invalid_street: "Invalid street (max. 200 characters)",
+          invalid_phone: "Invalid phone number. Please use the format +49931123 or 12344 without - or /",
+          invalid_description: "Invalid description (max. 2000 characters)",
+          invalid_name: "Invalid laboratory name (max. 200 characters)",
+          invalid_number: "Invalid number",
+          invalid_search: "Invalid search parameter",
+          invalid_role: "Invalid search type",
+          invalid_skills: "Invalid skills",
+          invalid_equipment: "Invalid equipment",
+          invalid_advice: "Invalid request",
+          invalid_consent: "Invalid consent",
+          invalid_url: "Invalid URL, has to start with http:// or https:// ",
+          passwordMatch: "Passwords don't match!"
+        },
+        not_authorized: "Not authorized!",
+        invalid_location: "Invalid address!",
+        internal_error: "Something went wrong...",
+        bad_request: "Invalid request!",
+        invalid_password: "Invalid password!",
+        invalid_login: "Invalid login details!",
+        existing_user: "User already exists!",
+        false_consent: "Whithout consent, we're not allowed to process your data.",
+        no_results: "No results were found...",
+        tokenTooOld: "Please reset the password again, the reset link expired.",
+        contact_support: "An error has occurred. Please contact support@labhive.de and apologize the inconvenience!",
```

Comment:
"Please contact support@labhive.de. We apologize for the inconvenience!"

(The way it is now makes it seem like a command for the client to apologize in the email they send ;)